### PR TITLE
Website/ Performance sprint

### PR DESCRIPTION
**Pull request by DavidBachmann**
# Performance sprint

## Main issues:
[x] Hero slider is way too heavy
[x] Hero causes page jumping
[ ] Initial JS payload is over 600kb

### Hero animations are being temporarily removed to establish a baseline performance and to improve the homepage ASAP. Animations will be put back in using a different strategy soon.

Animation strategy going forward:
- Keep animation files local and pick them by ID from Contentful if they have to be picked.
- Serve a basic SVG version and enhance it with a dynamically loaded lottie-web canvas implementation.

### CODE REVIEW

File: apps/web/components/FrontpageTabs/illustrations/ManOnBenchIllustration.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+
+function ManOnBenchIllustration() {
```
Comment (arnifreyrs) [inline apps/web/components/FrontpageTabs/illustrations/ManOnBenchIllustration.tsx:3]: other projects need these illustrations. I suggest that these illustrations are moved to a shared library.

**Comment by stefanhar** [comment]
Did we view alternative methods that allow us to keep the animations e.g. loading them lazily or load them as assets instead of string?

File: apps/web/components/FrontpageTabs/illustrations/ManOnBenchIllustration.tsx (hunk 1 above)
Comment (DavidBachmann) [reply apps/web/components/FrontpageTabs/illustrations/ManOnBenchIllustration.tsx:3]: Good call

**Comment by DavidBachmann** [comment]
> Did we view alternative methods that allow us to keep the animations e.g. loading them lazily or load them as assets instead of string?

This Slider change will increase performance ASAP because it's unacceptable right now — it will **also** give us a baseline to try to incorporate the animations performantly while taking the time we need to do so.

**Review by stefanhar — APPROVED** [review]

**Review by darrikonn — DISMISSED** [review]

**Comment by DavidBachmann** [comment]
Closed in favor of #1576

**Pull request closed without merging**